user_guide/data_management_view.rst

```diff
@@ -100,6 +100,21 @@ If there are more than four actions available for a record, they will be collect
 
 |
 
+
+Review the Record History
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If this is permitted to the user (the :ref:`Data Audit capabilty <admin-capabilities-data-audit>` has been enabled for 
+at least one of the user's :ref:`roles <user-guide-user-management-permissions-basic>`, in the top right corner of the 
+View page, there will be a link to the history of operations performed with the record.
```

Comment:
Edit: If the user is allowed to do this (i.e., the :ref:`Data Audit capability <admin-capabilities-data-audit>` has been enabled for at least one of the user's :ref:`roles <user-guide-user-management-permissions-basic>`), there will be a link to the history of operations performed with the record in the top right corner of the View page.